### DISCUSSION TITLE
How can I do a fresh installation of Wasp using version 0.13?

### DISCUSSION THREAD

**Discussion by Johan-IDX** [wasp-lang/wasp#2209] [Archived]
I want to perform a fresh installation of that specific version. What would be the way to do it? Thanks!

**Comment by Martinsos** [comment]
Hey! If you have `wasp` installed you can just run `wasp version` for details.
Here is instruction: `curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s -- -v 0.13.2`.
Closimg because this should be it but let me know if you need anything else!
